### DISCUSSION TITLE
problem with lightroom 13 and midi2LR

### DISCUSSION THREAD

**Discussion by fotonty** [rsjaffe/MIDI2LR#748] [Problems]
HI, i have just upgrade to lightroom 13 and midi2lr is just un-usable, it slow down everything..
If i Disable it everything works fine.
someone have a solution?

**Comment by antzpantz** [comment]
Odd... I'm using MIDI2LR now with newly updated Lightroom 13 and it seems
fine?

**Reply by fotonty** [reply]
After a long afternoon trying everything i think it's a problem with my keyboard.
Just to give you an example:
if i use the controller to make some change then i use the USB keyboard arrow to go to the next image, lightroom goes crazy and jump to 2 photos .
if i only use the controller everything is fine.
thank you!
